runtime-watcher/internal/watchermetrics/metrics.go

```diff
@@ -68,6 +77,18 @@ func (w *WatcherMetrics) UpdateRequestDuration(duration time.Duration) {
 	w.requestDurationGauge.Set(duration.Seconds())
 }
 
+func (w *WatcherMetrics) UpdateFipsMode() {
+	fipsMode := 0
```

Comment:
Consider to make it as global on top of the file as `const` just for a fixed values and readability:
```
const (
    FipsModeOff  = 0
    FipsModeOn   = 1  // includes "on" and "debug"
    FipsModeOnly = 2
)
```